base/loading.jl

```diff
@@ -1828,7 +1828,7 @@ function compilecache_path(pkg::PkgId;
     path = nothing
     isnothing(sourcepath) && error("Cannot locate source for $(repr("text/plain", pkg))")
     for path_to_try in cachepaths
-        staledeps = stale_cachefile(sourcepath, path_to_try, ignore_loaded = true, requested_flags=flags)
+        staledeps = stale_cachefile(sourcepath, path_to_try; ignore_loaded, requested_flags=flags)
```

Comment:
IIUC, this was to allow the user to compute the best pkg to load for `compilecache_path`, even if it is currently loaded at a different version?

Reply:
Or maybe it was just missed originally? https://github.com/JuliaLang/julia/pull/50218/commits/c24136d7d558535b2673b651ea5f46ac41761598

Reply:
I think it could've been. It's not mentioned in the PR https://github.com/JuliaLang/julia/pull/50218